src/build/__main__.py

```diff
@@ -159,7 +158,8 @@ def main_parser():  # type: () -> argparse.ArgumentParser
         '--config-setting',
         '-C',
         action='append',
-        help='pass option to the backend',
+        help='pass options to the backend.  Options which begin with a hyphen must be in the form of '
+        '`--config-setting=--opt` or `-C--opt`',
```

Comment:
Maybe add the optional `=value` here?

Reply:
done

Reply:
Well, I was thinking `-C--opt(=value)`, sorry for not being clear :sweat_smile: